package.json

```diff
@@ -11,11 +11,12 @@
   "dependencies": {
     "@mdi/font": "^3.6.95",
     "core-js": "^3.4.4",
-    "panzoom": "^9.2.1",
     "jslint": "^0.12.1",
+    "panzoom": "^9.2.1",
     "register-service-worker": "^1.6.2",
     "roboto-fontface": "*",
     "vue": "^2.6.10",
+    "vue-swatches": "^1.0.4",
```

Comment:
It looks like you didn't end up using this library? If so, can you remove it from here?